packages/dapp/src/components/listinglist/ListingsInProgress.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from "react";
+import { Set } from "immutable";
+import { Tabs } from "../tabs/Tabs";
+import { Tab } from "../tabs/Tab";
+
+import ListingList from "./ListingList";
+import { connect } from "react-redux";
+import { State } from "../../reducers";
+
+export interface ListingProps {
+  applications: Set<string>;
+  readyToWhitelistListings: Set<string>;
+  inChallengeCommitListings: Set<string>;
+  inChallengeRevealListings: Set<string>;
+  awaitingAppealRequestListings: Set<string>;
+  awaitingAppealJudgmentListings: Set<string>;
+  awaitingAppealChallengeListings: Set<string>;
+  appealChallengeCommitPhaseListings: Set<string>;
+  appealChallengeRevealPhaseListings: Set<string>;
+  resolveChallengeListings: Set<string>;
+  resolveAppealListings: Set<string>;
+}
+
+class ListingsInProgress extends React.Component<ListingProps> {
+  public render(): JSX.Element {
+    const applications = this.props.applications;
```

Comment:
should these be represented in the reducers?  Like instead of merging them here, get rid of the smaller categories and replace them with meaningful sets?

Reply:
I'd rather keep the small categories at least for now. Don't think we're even sure what exactly the bigger categories would be, this makes it a bit easier to change things around?